docs/Extensions.md

```diff
@@ -37,6 +37,58 @@ You can see the [vscode-xml-maven](https://github.com/angelozerr/vscode-xml-mave
 
 ![demo of vscode xml maven suggesting different scopes for a dependency](./images/vscode-xml-maven.gif)
 
+### Debugging your extension
+
```

Comment:
I will add a little introduction on how to register the LemMinx extension:

 * with a settings
 * with package.json from a vscode extension.

LemMinx extension must be compiled.

Reply:
I think I will add a section on "Running your Java LemMinX extension", then say to be sure to follow those steps before debugging.